logstash.go

```diff
@@ -74,6 +74,8 @@ func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
 			Hostname: m.Container.Config.Hostname,
 		}
 
+		tags := GetContainerTags(m.Container.Config.Env)
```

Comment:
Yeah, please take a look if you want. Otherwise try something as I suggested, checking a map key for existence is pretty fast and would be ok inside the loop.